api/filter/http/transcoder.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+// This is a filter which allows a RESTful JSON API client to send requests
+// to Envoy over HTTP and get proxied to a gRPC service. The HTTP mapping
+// for the gRPC service has to be defined by custom options, defined in
+// https://cloud.google.com/service-management/reference/rpc/google.api#http
+message gRPCJsonTranscoder {
```

Comment:
Downvote, message names must begin with capital letters. This would be best written `GrpcJsonTranscoder`